_posts/2016-12-04-bachelorkolloq-mender.md

```diff
@@ -9,8 +9,8 @@ Das kommende Bachelorkolloquium bezieht sich auf eine Arbeit, für welche nicht
 Simone Mender arbeitete unter Betreuung von Prof. Kröninger am Materialprüfungsamt. Die Arbeit am Materialprüfungsamt ist für Medizinphysik-Studierende wie auch für Hauptfach-Studierende von Interesse.
 
 Am Materialprüfungsamt NRW wird das neuartige Dünnschicht-Thermolumineszenzdosimetersystem (TL-DOS) zur Messung der Tiefenpersonendosis entwickelt. Diese Dosimeter sollen zukünftig die herkömmlichen Filmdosimeter ersetzen. Um eine Strahlenexposition oberhalb des natürlichen Untergrundes zu registrieren, müssen Dosimeter eine Dosis von etwa 0,1 mSv sicher nachweisen können.
-Ziel dieser Arbeit ist es, die Nachweisgrenze der aktuellen Dosimeter des TL-DOS Projekts zu bestimmen und Möglichkeiten zu finden, diese zu senken. Mit dem Ergebnis ihrer Arbeit leistet Simone einen wichtigen Beitrag zur Vorbereitung für das Bauartzulassungsverfahren.
```

Comment:
Simone does not want the sentence starting with "Mit dem Ergebnis ihrer Arbeit", so I deleted it.